ui/raidboss/data/04-sb/eureka/eureka_hydatos.js

```diff
@@ -951,7 +951,7 @@
       id: 'BA Ozma Adds',
       regex: Regexes.ability({ source: 'Cloudlarker', id: '37B0', capture: false }),
       regexDe: Regexes.ability({ source: 'Wolkenlauerer', id: '37B0', capture: false }),
-      regexFr: Regexes.ability({ source: 'Urolithe De L\'arsenal', id: '37B0', capture: false }),
+      regexFr: Regexes.ability({ source: 'Rôdeur Des Nuages', id: '37B0', capture: false }),
```

Comment:
Please don't touch this one. It does not correspond to Eureka, but to a mob of Hw. `rodeur des nuages` doesn't exist on ozma, I haven't found it anywhere. The add on Ozma is `Urolithe De L\'arsenal`
Here on xivapi the mob rodeur des nuages of Hw: https://xivapi.com/BNpcName/4608?pretty=true

Reply:
This is not the one from the trigger, the trigger uses an "untargetable" add that cast the skill

Check please your log if you can find it there

Reply:
I don't think so because in English the name of the add is cloudlarker. rodeur des nuages does not exist anywhere (mob, skill or other) on Ozma.

Reply:
The source is the add for me. The iD also corresponds with what I have pictured for you.